src/components/PomodoroTimer.tsx

```
@@ -293,6 +318,10 @@ const PomodoroTimer: React.FC<PomodoroTimerProps> = React.memo(({ settings }) =>
     );
   }
 
+  const handleDismissPermissionPrompt = useCallback(() => {
```

ESLint

[error] React Hook "useCallback" is called conditionally. React Hooks must be called in the exact same order in every component render.

(react-hooks/rules-of-hooks)